apps/site/lib/site_web/templates/schedule/_timetable.html.eex

```diff
@@ -83,7 +83,7 @@
               class: "js-tt-stop-name m-timetable__cell--first-column m-timetable__cell m-timetable__cell--" <>
                 cell_background
             ] do %>
-              <div class="m-timetable__row-header m-timetable__stop-name">
+              <div class="m-timetable__row-header m-timetable__stop-name notranslate">
```

Comment:
looks like this one works for the list on the side but the direction picker is also a part of this ticket

![image](https://user-images.githubusercontent.com/526017/160138688-2b9b9218-3ab9-4fe1-a85d-2219335ed846.png)

Reply:
Resolved this in the line diagram ticket since it uses the same direction picker component. I'll merge that one in first.

Reply:
@i0sea It's not the same direction picker component.